banana.csv

```diff
@@ -1,15 +0,0 @@
-buy,10
-supply,300
-buy,130
-buy,12
-supply,20
-buy,1
-supply,50
-supply,12
-supply,8
-buy,41
-supply,10
-buy,12
-buy,10
-buy,77
-supply,91
```

Comment:
I have just tested it locally. Everything works fine. You can add the content of this file back